gc.c

```diff
@@ -13626,189 +13634,191 @@ rb_raw_obj_info(char *buff, const int buff_size, VALUE obj)
+                                rb_id2name(me->called_id),
+                                METHOD_ENTRY_VISI(me) == METHOD_VISI_PUBLIC ?  "pub" :
+                                METHOD_ENTRY_VISI(me) == METHOD_VISI_PRIVATE ? "pri" : "pro",
+                                METHOD_ENTRY_COMPLEMENTED(me) ? ",cmp" : "",
+                                METHOD_ENTRY_CACHED(me) ? ",cc" : "",
+                                METHOD_ENTRY_INVALIDATED(me) ? ",inv" : "",
+                                me->def ? rb_method_type_name(me->def->type) : "NULL",
+                                me->def ? me->def->alias_count : -1,
+                                (void *)me->owner, // obj_info(me->owner),
+                                (void *)me->defined_class)); //obj_info(me->defined_class)));
+
+                        if (me->def) {
+                            switch (me->def->type) {
+                            case VM_METHOD_TYPE_ISEQ:
+                                APPENDF((BUFF_ARGS, " (iseq:%s)", obj_info((VALUE)me->def->body.iseq.iseqptr)));
+                                break;
+                            default:
+                                break;
+                            }
+                        }
+
+                        break;
+                    }
+            case imemo_iseq: {
+            const rb_iseq_t *iseq = (const rb_iseq_t *)obj;
+                    rb_raw_iseq_info(BUFF_ARGS, iseq);
+            break;
+            }
+                case imemo_callinfo:
+                    {
+                        const struct rb_callinfo *ci = (const struct rb_callinfo *)obj;
+                        APPENDF((BUFF_ARGS, "(mid:%s, flag:%x argc:%d, kwarg:%s)",
+                                rb_id2name(vm_ci_mid(ci)),
+                                vm_ci_flag(ci),
+                                vm_ci_argc(ci),
+                                vm_ci_kwarg(ci) ? "available" : "NULL"));
+                        break;
+                    }
+                case imemo_callcache:
+                    {
+                        const struct rb_callcache *cc = (const struct rb_callcache *)obj;
+                        VALUE class_path = cc->klass ? rb_class_path_cached(cc->klass) : Qnil;
+                        const rb_callable_method_entry_t *cme = vm_cc_cme(cc);
+
+                        APPENDF((BUFF_ARGS, "(klass:%s cme:%s%s (%p) call:%p",
+                                NIL_P(class_path) ? (cc->klass ? "??" : "<NULL>") : RSTRING_PTR(class_path),
+                                cme ? rb_id2name(cme->called_id) : "<NULL>",
+                                cme ? (METHOD_ENTRY_INVALIDATED(cme) ? " [inv]" : "") : "",
+                                (void *)cme,
+                                (void *)vm_cc_call(cc)));
+                        break;
+                    }
+            default:
+            break;
+            }
+        }
+        default:
+            break;
+        }
+    }
```

Comment:
Indents seem broken.

Reply:
I indented this `block` by one extra because I wrapped it in `if (obj_info_flags & OBJ_INOF_EXT)`. I can keep the indenting as-is if you prefer? Then it'll look like:

```
if (obj_info_flags & OBJ_INFO_EXT) {
switch(type) {
case T_NODE:
    ... etc
}
}
```

i.e. the if and the switch it contains will be indented the same. Is that what you want me to do?

Reply:
Seems inconsistent inside single block at least.
```C
        case T_NODE:
            UNEXPECTED_NODE(rb_raw_obj_info);
            break;
        case T_ARRAY:
                if (FL_TEST(obj, ELTS_SHARED)) {
                    // snip
                }
        // snip
            case T_SYMBOL: {
```
and
```C
        case T_IMEMO: {
                APPENDF((BUFF_ARGS, "<%s> ", rb_imemo_name(imemo_type(obj))));

            switch (imemo_type(obj)) {
            case imemo_ment:
                    {
                        // snip
                    }
            case imemo_iseq: {
            const rb_iseq_t *iseq = (const rb_iseq_t *)obj;
                    rb_raw_iseq_info(BUFF_ARGS, iseq);
            break;
            }
                case imemo_callinfo:
                    {
```